### DISCUSSION TITLE
Why are TM-30-18 Report Hue-Angle Bin Chart bar heights and numbers not matching?

### DISCUSSION THREAD

**Discussion by Paul-Sims** [colour-science/colour#861] [Q&A] [answered]
You can see this in the release notes and in every SPD I have run against this report script.

Example:
At this bullet item: "Implement support for ANSI/IES TM-30-18 Colour Rendition Report. (@enneract, @KelSolaar)" 
in https://github.com/colour-science/colour/releases

The bar heights and the numbers above them do not match in the Hue-Angle Bin plot.

Regards,

Paul

**Comment by thomasmansencal** [comment, marked as answer]
Hi Paul,

This is a small defect, the vertical offset `vo` is added to the value before plotting! Fix inbound!

**Reply by thomasmansencal** [reply]
This should be fixed now in develop!
